src/core/mustache/template_string.rs

```diff
@@ -92,16 +92,16 @@ mod tests {
 
     #[test]
     fn test_parse() {
-        let actual = TemplateString::parse("{{.env.TAILCALL_SECRET}}").unwrap();
-        let expected = Mustache::parse("{{.env.TAILCALL_SECRET}}").unwrap();
+        let actual = TemplateString::parse("{{.env.TAILCALL_LLM_API_KEY}}").unwrap();
+        let expected = Mustache::parse("{{.env.TAILCALL_LLM_API_KEY}}").unwrap();
```

Comment:
Are these changes necessary?

Reply:
No, I used apiKey field name earlier so renamed env also. Reverting this.